tensorflow_encrypted/protocol/pond.py

```diff
@@ -434,6 +434,29 @@ def transpose(self, x):
 
         return x_t
 
+    # see https://www.tensorflow.org/api_docs/python/tf/strided_slice for documentation on
+    # the arguments
+    def strided_slice(self, x, *args, **kwargs):
+        node_key = ('strided_slice', x)
+        x_t = _nodes.get(node_key, None)
+
+        if x_t is not None:
+            return x_t
+
+        if isinstance(x, PondPublicTensor):
```

Comment:
should we use the dispatch pattern from `pond`? maybe not enough types to be worth it

Reply:
Other unary ops do it like this.